_includes/content/environment-file.md

```diff
@@ -19,33 +19,46 @@ Use the following guidelines and sample file to define the environment variables
  [Felix configuration reference]({{site.baseurl}}/reference/felix/configuration).
 {% endif %}
 
+{% tabs %}
+  <label: Kubernetes datastore,active:true>
+  <%
 
-For an etcdv3 datastore set the following
+For a Kubernetes datastore (default) set the following:
 
 | Variable | Configuration guidance |
 |----------|------------------------|
-| {{datastore_type}} | Set to `etcdv3` |
-| {{etcd_endpoints}} | Comma separated list of etcdv3 cluster URLs, e.g. https://calico-datastore.example.com:2379 |
-| {{etcd_ca_file}} | Path to CA certificate to validate etcd's server cert.  Required if using TLS and not using a public CA. |
-| {{etcd_cert_file}}<br>{{etcd_key_file}} | Paths to certificate and keys used for client authentication to the etcd cluster, if enabled.   |
+| {{datastore_type}} | Set to `kubernetes` |
+| KUBECONFIG | Path to kubeconfig file to access the Kubernetes API Server |
 
 {% if include.install == "container" %}
-> **Note**: If using certificates and keys, you will need to volume mount them in the container and the path values above are *inside* the container.
+> **Note**: You will need to volume mount the kubeconfig file into the container at the locations specified by the paths mentioned above.
```

Comment:
since we're talking about a single volume mount here, it should be "location" not "locations".